gno.html.markdown

```diff
@@ -0,0 +1,533 @@
+---
+language: Gno
```

Comment:
you're missing the `filename` metadata field

Reply:
Hey @vendethiel, thanks for the comment.

I wasn't fully sure what that line was for, as the repo currently contains mainly html markdown files, without any other files. Can you clarify what this adding this line would do? 🙏🏻

Reply:
Sets the filename for the `Get the code:` button